x-pack/plugins/stack_connectors/public/connector_types/openai/connector.test.tsx

```diff
@@ -101,7 +101,7 @@ describe('ConnectorFields renders', () => {
       }));
       const { queryByTestId } = render(
         <ConnectorFormTestProvider connector={openAiConnector}>
-          <ConnectorFields readOnly={false} isEdit={false} registerPreSubmitValidator={() => {}} />
+          <ConnectorFields readOnly={false} isEdit registerPreSubmitValidator={() => {}} />
```

Comment:
this test was incorrect, _Does not render if **isEdit is true** and dashboardUrl is null_